lib/fathead/scala/parse.py

```diff
@@ -13,6 +13,23 @@ def get_all_files():
 			doc_files.append( "%s/%s" % (path, f))
 	return doc_files
 
+def extarct_article_text(article):
+	article_text = ""    
+	for i in article.children:
+        	if i.name == "pre" or i.name == "code":
+	        	article_text += "<pre>" + extarct_article_text(i) + "</pre>"
```

Comment:
@sam09 yes, exactly.

Inline snippets should be `<code>` to keep the flow of the text, but change the look of it.

`<pre><code>` is used to make a code block, which our front end will also apply syntax highlighting to